package.json

```diff
@@ -27,6 +27,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@d2l/aws-sdk": "^0.31.0",
```

Comment:
One issue here is that I’ve added this as a dependency (to check if the `docClient` instance is a `D2LDocumentClient`) but it’s a private repo, and this repo is public

Reply:
Maybe leave the dependency out. You can always just check if doc client has the expected property